network/model.go

```diff
@@ -1,6 +1,9 @@
 package network
 
-type CreatePlayerModel struct {
-	Username    string `json:"username"`
-	DisplayName string `json:"displayName"`
+import "github.com/joshprzybyszewski/cribbage/model"
+
+type CreateInteractionRequest struct {
+	PlayerID model.PlayerID `json:"playerID"`
+	Mode     string         `json:"mode"`
+	Info     interface{}    `json:"info"`
```

Comment:
:oof: also, binding an interface could get tricky. we could/should add a field for each mode on this request struct instead of allowing `Info` :feelsbatman: